inc/admin.php

```diff
@@ -80,7 +80,17 @@ function settings_screen() {
 	}
 ?>
 <div class="wrap">
-<?php if ( ! empty( $errors ) ) : ?>
+
+	<?php if ( ads_txt_exists() ) : ?>
+		<div class="notice notice-error adstxt-notice existing-adstxt">
+			<p><strong><?php echo esc_html_e( 'Existing Ads.txt file found', 'ads-txt' ); ?></strong></p>
+			<p><?php echo esc_html_e( 'You will need to rename or remove the existing ads.txt file before you will be able to see any changes you make to ads.txt inside the WordPress admin.', 'ads-txt' ); ?></p>
+
+			<p><?php echo esc_html_e( 'Removed the existing ads.txt but are still seeing this warning?', 'ads-txt' ); ?> <a class="ads-txt-rerun-check" href="#"><?php echo esc_html_e( 'Re-run the check now', 'ads-txt' ); ?></a></p>
```

Comment:
semantically this should not be an `a` href link, rather it should be a button. (see https://blogs.ancestry.com/ancestry/2014/09/02/buttons-vs-anchors/)
```suggestion
			<p><?php echo esc_html_e( 'Removed the existing ads.txt but are still seeing this warning?', 'ads-txt' ); ?> <button class="ads-txt-rerun-check"><?php echo esc_html_e( 'Re-run the check now', 'ads-txt' ); ?></button></p>
```